yarn.lock

```diff
@@ -6384,7 +6391,7 @@ cross-spawn@6.0.5, cross-spawn@^6.0.0, cross-spawn@^6.0.5:
     shebang-command "^1.2.0"
     which "^1.2.9"
 
-cross-spawn@^5.0.1:
+cross-spawn@^5.0.1, cross-spawn@^5.1.0:
```

Comment:
I just noticed that running a `yarn install` on my local reverts all of these lock file changes. Did you upgrade some other dependencies in this scope by mistake?

Reply:
Do you mean when you check this branch out locally?

Reply:
Fixed.